eng/build.ps1

```diff
@@ -96,6 +96,7 @@ foreach ($argument in $PSBoundParameters.Keys)
     "verbosity"              { $arguments += " -$argument " + $($PSBoundParameters[$argument]) }
     "configuration"          { $configuration = (Get-Culture).TextInfo.ToTitleCase($($PSBoundParameters[$argument])); $arguments += " -configuration $configuration" }
     "arch"                   { $arguments += " /p:TargetArchitecture=$($PSBoundParameters[$argument])" }
+    "testnobuild"                   { $arguments += " /p:VSTestNoBuild=true" }
```

Comment:
```suggestion
    "testnobuild"            { $arguments += " /p:VSTestNoBuild=true" }
```

Reply:
Adding this change to the PR that I split off from this - https://github.com/dotnet/aspire/pull/8333